File: Python/백준/Silver/29378. Гарри Поттер и нос Волан－де－Морта/Гарри Поттер и нос Волан－де－Морта.py
a,b = map(int,input().split())
aa = []
cnt = 0
for i in range(a):
    aa.append(input())
for i in range(a):
    for j in range(b-1):
        if aa[i][j]==aa[i][j+1] and aa[i][j]=='.':
            cnt+=1
for i in range(b):
    for j in range(a-1):
        if aa[j][i]==aa[j+1][i] and aa[j][i]=='.':
            cnt+=1
print(cnt)

File: C#/백준/Silver/29378. Гарри Поттер и нос Волан－де－Морта/Гарри Поттер и нос Волан－де－Морта.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int a = aa[0];
            int b = aa[1];
            int cnt = 0;
            string[] s = new string[a];
            for (int i=0;i<a;i++){
                s[i] = Console.ReadLine();
            }
            for (int i=0;i<a;i++){
                for (int j=0;j<b-1;j++){
                    if(s[i][j]==s[i][j+1] && s[i][j]=='.'){cnt++;}
                }
            }
            for (int i=0;i<b;i++){
                for (int j=0;j<a-1;j++){
                    if(s[j][i]==s[j+1][i] && s[j][i]=='.'){cnt++;}
                }
            }
            Console.WriteLine(cnt);
        }
    }
}